dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -256,20 +235,19 @@ public Response generateIntegrityData(@Context HttpServletRequest request)  {
     /**
      * Checks if the generation of Integrity Data is done.
      * If FINISHED, returns a zip with the data
-     * if PROCESSING, returns HttpStatus.SC_PROCESSING
+     * if PROCESSING, returns HttpStatus.SC_ACCEPTED
      * if ERROR, returns HttpStatus.SC_INTERNAL_SERVER_ERROR, including the error message
      *
-     * Usage: /getdata
+     * Usage: /integrityData
      *
      */
-    @POST
-    @Path("/{requestId}/status")
+    @GET
+    @Path("/{requestId}/integrityData")
```

Comment:
is this safe? Do we need to change the call at JSP or angular?

Reply:
well probably b/c it is not changing only the path, but also the verb

Reply:
This endpoint is meant to be consumed by backend http clients to sync the integrity check between dotcms instances.
Although, at `core` I didn't find any place where we consume this endpoint from a UI. I will have to ask @fmontes to check it on `core-web`.

Reply:
According to @fmontes there is no impact on `core-web`.

Reply:
I search for it and I don't think we're using it in core-web